test/runtests.jl

```diff
@@ -49,6 +49,9 @@ end
 @safetestset "Neural Snow model tools tests" begin
     include("standalone/Snow/tool_tests.jl")
 end
+@safetestset "Snow integrated content" begin
```

Comment:
I think this is a little more clear:
```suggestion
@safetestset "Snow integrated water, energy content" begin
```